app/translations/en_US.json

```diff
@@ -157,10 +158,9 @@
   "RESULTS": "Results",
   "NO_RESULTS": "Unfortunately, there were no results matching your filters.",
 
-  "INTERP_INTRO": "Here you can visualize senseBox measurement data.",
+  "INTERP_INTRO": "Here you can visualize senseBox measurement data for areas. The <i>Inverse Distance Weighting</i> algorithm is used.",
```

Comment:
@noerw could you also include the wikipedia link to the english article for IDW https://en.wikipedia.org/wiki/Inverse_distance_weighting